src/libraries/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -218,6 +219,7 @@ public static partial class DataReaderExtensions
         public static bool IsDBNull(this System.Data.Common.DbDataReader reader, string name) { throw null; }
         public static System.Threading.Tasks.Task<bool> IsDBNullAsync(this System.Data.Common.DbDataReader reader, string name, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
+#nullable disable
```

Comment:
I can't comment on it, but the `T GetFieldValue<T>` method above... are any Ts possible that would result in a null being returned?

Reply:
No, not to the best of my knowledge - System.Data is quite consistent with representing database nulls via DBNull.Value. I'll take another look at the SqlClient implementation to see if I'm wrong here and there's some edge cases.

Reply:
I kept it as `T GetFieldValue<T>` when null-annotating MySqlConnector.

Reply:
I've taken a look inside SqlClient, and it does confirm my understanding that null is never returned. It seems possible to get a null by by registering a UDT whose null representation is a null, but that's quite contrived and is probably a case of API misuse.